src/client/Health/NodeView.jsx

```diff
@@ -6,6 +6,10 @@ import ReactFlow from 'react-flow-renderer'
 import { ConfigContext } from '../context/configContext.jsx'
 import PointsButton from '../South/PointsButton.jsx'
 import Modal from '../components/Modal.jsx'
+import logo from './OIBus.png'
+import mqtt from './Mqtt.png'
+import ProtocolsImages from '../South/ProtocolsImages.jsx'
+import ApisImages from '../North/ApisImages.jsx'
```

Comment:
After a discussion with JF, we don't need these files: instead of importing theses files, import Protocols.jsx and Apis.jsx files

Then, on each protocol/application schema.jsx file, import the specific protocol/application file and store it in a new 'image' property of the schema object

Back to the NodeView file: you can find this new image property the same way we were getting the right image in this PR